src/Http/Controllers/Api/Geojson/VolumeImagesController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Biigle\Modules\Geo\Http\Controllers\Api\Geojson;
+
+use DB;
+use Biigle\Volume;
+use Biigle\Http\Controllers\Api\Controller;
+use League\Flysystem\FileNotFoundException;
+
+class VolumeImagesController extends Controller
+{
+  public function index($id){
```

Comment:
You could add an APIdoc comment for this function ([example](https://github.com/biigle/volumes/blob/0f663af6f1ec18dbe82b1c4ee379fa6aa1e86edd/src/Http/Controllers/Api/VolumeImageLabelsController.php#L10-L48)).

Reply:
Also, in the coding style used for the BIIGLE codebase, there is a newline between the function declaration and the curly bracket:

```php
public function index($id)
{
// ...
}
```